options/locale/locale_en-US.ini

```diff
@@ -1090,7 +1090,7 @@ settings.event_issue_comment_desc = Issue comment created, edited, or deleted.
 settings.event_release = Release
 settings.event_release_desc = Release published, updated or deleted in a repository.
 settings.event_pull_request = Pull Request
-settings.event_pull_request_desc = Pull request opened, closed, reopened, edited, assigned, unassigned, label updated, label cleared or synchronized.
+settings.event_pull_request_desc = Pull request opened, closed, reopened, edited, approved, rejected, assigned, unassigned, label updated, label cleared or synchronized.
```

Comment:
Please also note that this hook also fires when a pull request recieves a comment.

Reply:
Oops. would update.